charts/shoot-core/components/charts/monitoring/charts/kube-state-metrics-shoot/templates/rbac.yaml

```diff
@@ -52,3 +52,5 @@ roleRef:
 subjects:
 - kind: User
   name: garden.sapcloud.io:monitoring:kube-state-metrics
+- kind: User
+  name: core.gardener.cloud:monitoring:kube-state-metrics
```

Comment:
Sorry, can you rename `core.gardener.cloud:monitoring:kube-state-metrics` to `gardener.cloud:monitoring:kube-state-metrics`? The `core.` prefix was a mistake in the first place.

Reply:
Can do, but should we then also change all the other CN/O names like `core.gardener.cloud:monitoring` accordingly?

Reply:
Yep, sure, thank you!

Reply:
Done, PTAL